assets/html/index.html

```diff
@@ -1 +1,22 @@
-
+<html>
+  <head>
+    <title>Challenge One</title>
+  </head>
+  <body>
+    <h1 align="center">This is Chris Featley's page</h1>
+    <p align="center">This should be a photo of Chris</p>
+    <!--I realise I need to actually add a photo here and then do img src -->
+    <p align="center">My Hobbies</p>
+    <ul align="center">
+      <li>hobby 1</li>
+      <li>hobby 2</li>
+      <li>hobby 3</li>
```

Comment:
the bullet points are left aligned, but the text is centered? doesn't look like the photo

Reply:
all centre alignment